portfolio:desktop version

joffreynk opened the pull request:
### **portfolio:desktop version**

- add web components
- design the appearance
- test linters errors
- the project is focused on responsiveness on mobile and desktop screens
- design of appearance changes based on the size of the device

Commit:
adding images and starting header in index.html

joffreynk committed:
finalize html code and do some styling

Commit:
adding styles on about section

joffreynk committed:
styling all componeents

Commit:
finalize with styles

Commit:
adjusting linters

Commit:
correcting indentations

sja-thedude commented on index.html:
```diff
@@ -0,0 +1,186 @@
+          <p class="connected">LET`S CONNECT</p>
+          <ul>
+            <li><a href="#"> <img src="./images/header/twitter.svg" alt="twitter"> </a></li>
+            <li><a href="#"> <img src="./images/header/Vector.svg" class="in" alt="i"><img class="in" src="./images/header/Vector2.svg" alt="n"> </a></li>
```
MOBILE VERSION:

- [ ] Kindly make sure that the mobile version works as per the Figma design.
- [ ] Kindly make sure that the first page works as per the mobile version. Right now only the desktop version is seen in the place of the first page.
- [ ] Kindly reduce the extra space to the right and left sides from all the sections.
- [ ] Kindly make sure to follow the same images and spacing for all the cards.
- [ ] Kindly make sure that the image is placed first for the mobile version for all the cards.
- [ ] Kindly make sure to reduce the extra spacing from about section, all cards.
- [ ] Kindly follow the same background image for the first page and contact form as per the Figma design.
- [ ] Kindly make sure to write the same placeholder text as per the Figma design.
- [ ] Kindly make sure you follow the same width from the contact section paragraph, and input fields and place the button correctly as per the Figma design.

Your design:

https://user-images.githubusercontent.com/70269507/167120218-f015975a-c0e2-447c-b128-36e357df3d3d.mov

Figma design:

https://user-images.githubusercontent.com/70269507/167120514-47ca1e2a-3557-420a-a272-d1d36099a958.mov

zieeco replied:
**[REQUIRED CHANGED]**


- [ ] Hey, you've done a great job with your design, however according to the previous reviewer, you mobile version has a horizontal scroll. you need to address it. 
- One of the reasons is from your `header` [here](https://github.com/JoffreyNK/portfolio-desktop/blob/0d615afc45e85fb8e848c92bf26f89e641d547d7/desktop.css#L1-L21)
The width is too big and fixed, also you've used the wrong approach to adding the background image.
Instead, download the whole image from Figma
When implementing responsive design, your width should be in percentage or view width.
Try playing around the width value..

sja-thedude commented on index.html:
```diff
@@ -0,0 +1,186 @@
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button>see project</button>
```
- [ ] Kindly add button interactions for all the buttons in the project such as hover, pressed, enabled and disabled.

<img width="466" alt="Screenshot 2022-05-06 at 2 29 48 PM" src="https://user-images.githubusercontent.com/70269507/167123401-b6d6d807-107f-4e80-9cc0-3bda5e667d4b.png">

sja-thedude commented on index.html:
```diff
@@ -0,0 +1,186 @@
+            <li><a href="#"> <img src="./images/header/twitter.svg" alt="twitter"> </a></li>
+            <li><a href="#"> <img src="./images/header/Vector.svg" class="in" alt="i"><img class="in" src="./images/header/Vector2.svg" alt="n"> </a></li>
+            <li><a href="#"> <img src="./images/header/medium.svg" alt="medium"> </a></li>
+            <li><a href="#"> <img src="./images/header/github.svg" alt="github"> </a></li>
+            <li><a href="#"> <img src="./images/header/angellist.svg" alt="angellist"> </a></li>
+          </ul>
+      </section>
+    </header>
+    <main>
+      <section>
+        <div class="experience-image">
+          <img src="./images/post/tonic.svg" alt="" />
+        </div>
+        <div class="experience-content">
+          <h2>Tonic</h2>
+          <ul>
+            <li>CANOPY</li>
+            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
+            <li>Back End Dev</li>
+            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
```
DESKTOP VERSION:

- [ ] Kindly make sure that you follow all the card images as given in the Figma design for the desktop version.
- [ ] Kindly make sure to reduce the space of the left and right sides of all the sections as per the Figma design.
- [ ] Kindly make sure the background image is as per the Figma design of the first section (intro).
- [ ] Kindly make sure to reduce the space between images and text in all the cards.
- [ ] Kindly make sure to fix the button colour for the entire project.
- [ ] Kindly make sure to add the same text for the placeholder as per the given Figma design for the contact section of the desktop version.
- [ ] Kindly make sure that the about section has the correct margin and padding to the left and right as per the Figma design. Also, make sure that there is no space at the bottom of the about section as per the Figma design.

- Your design:

https://user-images.githubusercontent.com/70269507/167123376-1c4fcfed-de87-4492-846d-d918644a56ff.mov

- Figma design:

https://user-images.githubusercontent.com/70269507/167123421-f867b085-9172-4bcc-a1ae-6f567d73bef0.mov

zieeco replied:
**[REQUIRED CHANGED]**

- [ ] Great making some adjustments as required by the previous reviewer however, the main issue with your design is the wrong usage of the `width property`. You've used values that are too big. please, kindly check where you've used it wrongly and reduce them in your
- `main, p, .main section, `, 

```
main section {
    height: 496px;
    width: 1156px;
    margin: 142px 142px 0 142px;
    display: flex;
    align-items: center;
    background-color: #fff;
    border: 1px solid #dfe1e6;
    border-radius: 24px;
    gap: 24px;
    padding: 24px;
}
```
- This is just a few. Kindly check all width property to adjust them

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @JoffreyNK, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![768070f3-bc9a-4aba-b8bc-25793e00ea1e_text](https://user-images.githubusercontent.com/70269507/159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff.gif)

## Highlights
✔️ Good PR Title
✔️ All linters are passing

## Required 
- [ ] Kindly make sure that your mobile version works perfectly fine. Right now it is not the same as Figma's design.
- [ ] Kindly make sure to edit your read me file with the project description, setup details and your contact information.
- [ ] _Check the comments under the review._

## Optional suggestions
- [ ] Kindly add a more descriptive description to the pull request.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

joffreynk committed:
style mobile version

joffreynk committed:
correct linters

joffreynk committed:
correct linters again

joffreynk committed:
remove linters errors

zieeco reviewed (changes requested):
Hi @JoffreyNK 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JoffreyNK commented:
Hello,
Figma doesn't allow me to download the whole background and I have used the
width which was used there
please, be clear on these requirements so that I can submit the project
before the deadline

On Sat, May 7, 2022 at 4:02 AM zieeco ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @JoffreyNK <https://github.com/JoffreyNK>
>
> Great work on making the changes requested by the previous reviewer 👏
> You've done well implementing some of the changes, however, there are some
> issues that you still need to work on to go to the next project, but you
> are almost there!
> To highlight 🎯
>
>    - No linter errors.✔️
>    - Nice design. ✔️
>    - Descriptive PR. ✔️
>
> Required Changes ♻️
>
> *Check the comments under the review.*
>
>
> <https://camo.githubusercontent.com/aa9137eba4285ba68320a19592e5e655279c09be5c8fc2fb7ef499d61cf26010/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f3855475651455370444d4d516c37796433462f67697068792e676966>
> Optional suggestions
>
>    - N/A
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#discussion_r867285953>
> :
>
> > +          <li><a href="#">Portfolio</a></li>
>
> +          <li><a href="#">about</a></li>
>
> +          <li><a href="#">contact</a></li>
>
> +        </ul>
>
> +      </nav>
>
> +      <section>
>
> +        <h1 class="mainheader">I`m Joffrey NKESHIMANA,</h1>
>
> +        <h2 class="mainheader"> Glad to see you!</h2>
>
> +          <p>
>
> +            I`m a software developer! I can help you building a product, feature
>
> +            or website look through some of my work and experience! If you like
>
> +            what you see and have a project you need coded, don`t hesitate to
>
> +            contact me.
>
> +          </p>
>
> +          <p class="connected">LET`S CONNECT</p>
>
> +          <ul>
>
> +            <li><a href="#"> <img src="./images/header/twitter.svg" alt="twitter"> </a></li>
>
> +            <li><a href="#"> <img src="./images/header/Vector.svg" class="in" alt="i"><img class="in" src="./images/header/Vector2.svg" alt="n"> </a></li>
>
>
>
>    - Hey, you've done a great job with your design, however according to
>    the previous reviewer, you mobile version has a horizontal scroll. you need
>    to address it.
>    - One of the reasons is from your header here
>    <https://github.com/JoffreyNK/portfolio-desktop/blob/0d615afc45e85fb8e848c92bf26f89e641d547d7/desktop.css#L1-L21>
>    The width is too big and fixed, also you've used the wrong approach to
>    adding the background image.
>    Instead, download the whole image from Figma
>    When implementing responsive design, your width should be in
>    percentage or view width.
>    Try playing around the width value..
>
> ------------------------------
>
> In index.html
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#discussion_r867287680>
> :
>
> > +            <li><a href="#"> <img src="./images/header/github.svg" alt="github"> </a></li>
>
> +            <li><a href="#"> <img src="./images/header/angellist.svg" alt="angellist"> </a></li>
>
> +          </ul>
>
> +      </section>
>
> +    </header>
>
> +    <main>
>
> +      <section>
>
> +        <div class="experience-image">
>
> +          <img src="./images/post/tonic.svg" alt="" />
>
> +        </div>
>
> +        <div class="experience-content">
>
> +          <h2>Tonic</h2>
>
> +          <ul>
>
> +            <li>CANOPY</li>
>
> +            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
>
> +            <li>Back End Dev</li>
>
> +            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
>
> +            <li>2015</li>
>
> +          </ul>
>
> +          <p>
>
> +            A daily selection of privately personalized reads; no accounts or
>
> +            sign-ups required.
>
> +          </p>
>
> +          <ul class="languages">
>
>
>
>    - Great making some adjustments however, the main issue with your
>    design is the wrong usage of the width property. You've used values
>    that are too big. please, kindly check where you've used it wrongly and
>    reduce them in your
>    - main, p, .main section, ,
>
> main section {
>
>     height: 496px;
>
>     width: 1156px;
>
>     margin: 142px 142px 0 142px;
>
>     display: flex;
>
>     align-items: center;
>
>     background-color: #fff;
>
>     border: 1px solid #dfe1e6;
>
>     border-radius: 24px;
>
>     gap: 24px;
>
>     padding: 24px;
>
> }
>
>
>
>    - This is just a few. Kindly check all width property to adjust them
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#pullrequestreview-965274749>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQNBFQ7LCFXLKVOEJOUAMXTVIXFKJANCNFSM5VFHZGEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

JoffreyNK commented:
Hey,
Can you let me know width recommended? Because I implemented what according
to figma design

On Sat, May 7, 2022, 4:02 AM zieeco ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @JoffreyNK <https://github.com/JoffreyNK>
>
> Great work on making the changes requested by the previous reviewer 👏
> You've done well implementing some of the changes, however, there are some
> issues that you still need to work on to go to the next project, but you
> are almost there!
> To highlight 🎯
>
>    - No linter errors.✔️
>    - Nice design. ✔️
>    - Descriptive PR. ✔️
>
> Required Changes ♻️
>
> *Check the comments under the review.*
>
>
> <https://camo.githubusercontent.com/aa9137eba4285ba68320a19592e5e655279c09be5c8fc2fb7ef499d61cf26010/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f3855475651455370444d4d516c37796433462f67697068792e676966>
> Optional suggestions
>
>    - N/A
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#discussion_r867285953>
> :
>
> > +          <li><a href="#">Portfolio</a></li>
>
> +          <li><a href="#">about</a></li>
>
> +          <li><a href="#">contact</a></li>
>
> +        </ul>
>
> +      </nav>
>
> +      <section>
>
> +        <h1 class="mainheader">I`m Joffrey NKESHIMANA,</h1>
>
> +        <h2 class="mainheader"> Glad to see you!</h2>
>
> +          <p>
>
> +            I`m a software developer! I can help you building a product, feature
>
> +            or website look through some of my work and experience! If you like
>
> +            what you see and have a project you need coded, don`t hesitate to
>
> +            contact me.
>
> +          </p>
>
> +          <p class="connected">LET`S CONNECT</p>
>
> +          <ul>
>
> +            <li><a href="#"> <img src="./images/header/twitter.svg" alt="twitter"> </a></li>
>
> +            <li><a href="#"> <img src="./images/header/Vector.svg" class="in" alt="i"><img class="in" src="./images/header/Vector2.svg" alt="n"> </a></li>
>
>
>
>    - Hey, you've done a great job with your design, however according to
>    the previous reviewer, you mobile version has a horizontal scroll. you need
>    to address it.
>    - One of the reasons is from your header here
>    <https://github.com/JoffreyNK/portfolio-desktop/blob/0d615afc45e85fb8e848c92bf26f89e641d547d7/desktop.css#L1-L21>
>    The width is too big and fixed, also you've used the wrong approach to
>    adding the background image.
>    Instead, download the whole image from Figma
>    When implementing responsive design, your width should be in
>    percentage or view width.
>    Try playing around the width value..
>
> ------------------------------
>
> In index.html
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#discussion_r867287680>
> :
>
> > +            <li><a href="#"> <img src="./images/header/github.svg" alt="github"> </a></li>
>
> +            <li><a href="#"> <img src="./images/header/angellist.svg" alt="angellist"> </a></li>
>
> +          </ul>
>
> +      </section>
>
> +    </header>
>
> +    <main>
>
> +      <section>
>
> +        <div class="experience-image">
>
> +          <img src="./images/post/tonic.svg" alt="" />
>
> +        </div>
>
> +        <div class="experience-content">
>
> +          <h2>Tonic</h2>
>
> +          <ul>
>
> +            <li>CANOPY</li>
>
> +            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
>
> +            <li>Back End Dev</li>
>
> +            <li><img src="./images/post/Counter.svg" alt="counter" /></li>
>
> +            <li>2015</li>
>
> +          </ul>
>
> +          <p>
>
> +            A daily selection of privately personalized reads; no accounts or
>
> +            sign-ups required.
>
> +          </p>
>
> +          <ul class="languages">
>
>
>
>    - Great making some adjustments however, the main issue with your
>    design is the wrong usage of the width property. You've used values
>    that are too big. please, kindly check where you've used it wrongly and
>    reduce them in your
>    - main, p, .main section, ,
>
> main section {
>
>     height: 496px;
>
>     width: 1156px;
>
>     margin: 142px 142px 0 142px;
>
>     display: flex;
>
>     align-items: center;
>
>     background-color: #fff;
>
>     border: 1px solid #dfe1e6;
>
>     border-radius: 24px;
>
>     gap: 24px;
>
>     padding: 24px;
>
> }
>
>
>
>    - This is just a few. Kindly check all width property to adjust them
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#pullrequestreview-965274749>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQNBFQ7LCFXLKVOEJOUAMXTVIXFKJANCNFSM5VFHZGEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

zieeco commented:
@JoffreyNK, The Figma is just a template of how your portfolio should look. If strictly follow all the CSS properties used, you will end up not having a responsive design as in your case now. You need to be able to maneuver your way to make sure your portfolio is responsive. 

**NOTE: The Figma design itself is not responsive**

*[You could find this video useful](https://www.youtube.com/watch?v=0gBnRll5XgA)*

JoffreyNK commented:
you can help me by giving me dimensions I can use because I want it
approved before they close platform[image: changes.JPG]

from there I don't see what to change

On Sat, May 7, 2022 at 9:39 AM zieeco ***@***.***> wrote:

> @JoffreyNK <https://github.com/JoffreyNK>, The Figma is just a template
> of how your portfolio should look. If strictly follow all the CSS
> properties used, you will end up not having a responsive design as in your
> case now. You need to be able to maneuver your way to make sure your
> portfolio is responsive.
>
> *NOTE: The Figma design itself is not responsive*
>
> *You could find this video useful
> <https://www.youtube.com/watch?v=0gBnRll5XgA>*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#issuecomment-1120157172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQNBFQ4LNK7OT2VAJECTAOTVIYM4BANCNFSM5VFHZGEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

zieeco commented:
- Your width should be in the range of  `1 - 100%` at a maximum...
- You only need a fixed width 0f `170px` in your social media icons container
- The rest width of your page should be relative to its parent container.

*If you still find it difficult, schedule a zoom call and invite me*

JoffreyNK commented:
let us share from here: Joffrey NKESHIMANA is inviting you to a scheduled
Zoom meeting.

Topic: Joffrey NKESHIMANA - Salle de réunion personnelle

Join Zoom Meeting
https://microverse.zoom.us/j/8273390920?pwd=Tf8l-CUhCrMbWIdmlXHGEvk5UecZ_u.1

Meeting ID: 827 339 0920
Passcode: Microverse



On Sat, May 7, 2022 at 10:00 AM zieeco ***@***.***> wrote:

>
>    - Your width should be in the range of 1 - 100% at a maximum...
>    - You only need a fixed width 0f 170px in your social media icons
>    container
>    - The rest width of your page should be relative to its parent
>    container.
>
> *If you still find it difficult, schedule a zoom call and invite me*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#issuecomment-1120160327>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQNBFQ3UEJISEHAGCNNVQCTVIYPKLANCNFSM5VFHZGEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

joffreynk committed:
refactor styles

joffreynk committed:
resstructure index.html

joffreynk committed:
updates image folder

JoffreyNK commented:
https://microverse.zoom.us/j/8273390920?pwd=Tf8l-CUhCrMbWIdmlXHGEvk5UecZ_u.1


On Sat, May 7, 2022 at 10:10 AM Joffrey NKESHIMANA ***@***.***>
wrote:

> let us share from here: Joffrey NKESHIMANA is inviting you to a scheduled
> Zoom meeting.
>
> Topic: Joffrey NKESHIMANA - Salle de réunion personnelle
>
> Join Zoom Meeting
>
> https://microverse.zoom.us/j/8273390920?pwd=Tf8l-CUhCrMbWIdmlXHGEvk5UecZ_u.1
>
> Meeting ID: 827 339 0920
> Passcode: Microverse
>
>
>
> On Sat, May 7, 2022 at 10:00 AM zieeco ***@***.***> wrote:
>
>>
>>    - Your width should be in the range of 1 - 100% at a maximum...
>>    - You only need a fixed width 0f 170px in your social media icons
>>    container
>>    - The rest width of your page should be relative to its parent
>>    container.
>>
>> *If you still find it difficult, schedule a zoom call and invite me*
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/JoffreyNK/portfolio-desktop/pull/1#issuecomment-1120160327>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQNBFQ3UEJISEHAGCNNVQCTVIYPKLANCNFSM5VFHZGEA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

joffreynk committed:
add about section in html file

joffreynk committed:
update css file

joffreynk committed:
add images of about section

joffreynk committed:
add class for row reverse

joffreynk committed:
add media query

joffreynk committed:
add deskto images

joffreynk committed:
fix linters error

joffreynk committed:
fix linter errors

prolajumokeoni commented:
> https://microverse.zoom.us/j/8273390920?pwd=Tf8l-CUhCrMbWIdmlXHGEvk5UecZ_u.1 On Sat, May 7, 2022 at 10:10 AM Joffrey NKESHIMANA ***@***.***> wrote:
> […](#)
> let us share from here: Joffrey NKESHIMANA is inviting you to a scheduled Zoom meeting. Topic: Joffrey NKESHIMANA - Salle de réunion personnelle Join Zoom Meeting https://microverse.zoom.us/j/8273390920?pwd=Tf8l-CUhCrMbWIdmlXHGEvk5UecZ_u.1 Meeting ID: 827 339 0920 Passcode: Microverse On Sat, May 7, 2022 at 10:00 AM zieeco ***@***.***> wrote: > > - Your width should be in the range of 1 - 100% at a maximum... > - You only need a fixed width 0f 170px in your social media icons > container > - The rest width of your page should be relative to its parent > container. > > *If you still find it difficult, schedule a zoom call and invite me* > > — > Reply to this email directly, view it on GitHub > <[#1 (comment)](https://github.com/JoffreyNK/portfolio-desktop/pull/1#issuecomment-1120160327)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/AQNBFQ3UEJISEHAGCNNVQCTVIYPKLANCNFSM5VFHZGEA> > . > You are receiving this because you were mentioned.Message ID: > ***@***.***> >

Hi @JoffreyNK were you able to meet with the previous reviewer have your questions been answered?

prolajumokeoni commented on style.css:
```diff
@@ -0,0 +1,618 @@
+  margin-top: 48px;
+}
+
+.desktop-ul,
+.desktop-image {
+  display: none;
+}
```
### OPTIONAL
- [ ] On the desktop version, in your media queries, kindly make the navigation links visible 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_167](https://user-images.githubusercontent.com/69638013/167262204-ca49f569-1913-4c89-adc3-aab9cbcd5323.png)  | ![Screenshot_165](https://user-images.githubusercontent.com/69638013/167262210-89f38b04-9d99-444e-a4bc-4d3ad069d986.png)

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,244 @@
+            <h2 class="skill-title">Frameworks</h2>
+            <img src="./images/arrowright.svg" alt="arow-dropdown-button">
+          </li>
+          <li class="line"></li>
+          <li class="frames-lfs">
+            <h2 class="skill-title">Skills</h2>
+            <img src="./images/arrowright.svg" alt="arow-dropdown-button">
+          </li>
+          <li class="line"></li>
```
### OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_169](https://user-images.githubusercontent.com/69638013/167262322-1300f47b-1993-462d-bf7f-0768e4924450.png)  |  ![Screenshot_168](https://user-images.githubusercontent.com/69638013/167262352-a732db65-eb4d-4ccc-ba93-e035e7b24819.png)

- [ ] Kindly add the right border color so your design looks exactly like the figma design 👍🏿

prolajumokeoni reviewed (approved):
Hi @JoffreyNK 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Kindly edit your readme file to have a description and author social media links this will make your project more professional, don't forget to delete broken links  😉 
_Check comments under review_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
